app/src/test/java/com/example/habitsmasher/test/DaysTrackerUnitTest.java

```diff
@@ -0,0 +1,122 @@
+package com.example.habitsmasher.test;
+
+import com.example.habitsmasher.DaysTracker;
+import static org.junit.Assert.*;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DaysTrackerUnitTest {
+
+    private DaysTracker _tracker;
+    private DaysTracker _stringTracker;
+
+    @Before
+    public void setupTracker(){
+
+        _tracker = new DaysTracker();
+    }
+
+    @Test
+    public void testSetupStringInput(){
+        _stringTracker = new DaysTracker("MO TU WE TH");
+        assertTrue(_stringTracker.getMonday());
+        assertTrue(_stringTracker.getTuesday());
+        assertTrue(_stringTracker.getWednesday());
+        assertTrue(_stringTracker.getThursday());
+        assertFalse(_stringTracker.getFriday());
+        assertFalse(_stringTracker.getSaturday());
+        assertFalse(_stringTracker.getSunday());
+    }
+    @Test
+    public void testSetupNoStringInput(){
```

Comment:
Can we **rename** the tests in this class to follow the expected format?

Expected: `methodYouAreTesting_withSomeCondition_expectSomeValue`

So here, we could do something like: `constructor_withEmptyDateInput_expectAllDaysToBeFalse`

You can follow this pattern for the rest of the tests in this file!
- [ ] rename unit tests